sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/reflect/DoFnSignatures.java

```diff
@@ -230,6 +234,14 @@ public static ParameterDescription of(
     }
   }
 
+  /**
+   * Annotation for unit tests to disable the timer and state scope checking.
```

Comment:
Add details to another paragraph in this Javadoc specifying exactly why - because Mockito creates a subclass that duplicates the annotations in a way that would violate the validation if a user did it.